fastparquet/test/test_api.py

```diff
@@ -123,8 +124,8 @@ def test_attributes(tempdir):
     assert pf.columns == ['x', 'y', 'z']
     assert len(pf.row_groups) == 2
     assert pf.count == 4
-    assert fn == pf.info['name']
-    assert fn in str(pf)
+    assert join_path(fn) == pf.info['name']
+    assert join_path(fn) in str(pf)
```

Comment:
All the other tests were reverted so they continue to use windows' `os.sep`, except for this one.  Since the internal filenames are in standard form, the `fn` must be converted to standard form for these asserts to pass.

Reply:
agreed